docs/examples/ref.rb

```diff
@@ -17,18 +17,14 @@
 
 aws_security_group 'ref-sg1' do
   vpc 'ref-vpc'
-  inbound_rules [ { ports: 22, protocol: :tcp, sources: [ '0.0.0.0/0' ] } ]
+  inbound_rules '0.0.0.0/0' => 22
 end
 
 aws_security_group 'ref-sg2' do
   vpc 'ref-vpc'
 
-  inbound_rules [
-    {:ports => 2223, :protocol => :tcp, :sources => ['ref-sg1'] }
-  ]
-  outbound_rules [
-    {:ports => 2223, :protocol => :tcp, :destinations => ['ref-sg1'] }
-  ]
+  inbound_rules 'ref-sg1' => 2224
+  outbound_rules 2224 => 'ref-sg1'
```

Comment:
The old syntax still works; but here's how you do it in the new:

```ruby
outbound_rules { port_range: 2224, protocol: :udp } => 'ref-sg1'
inbound_rules 'ref-sg1' => { port_range: 2224, protocol: :udp }
```

Arrays work as well, on both sides:

```ruby
outbound_rules [22, 1024..2048] => [ '10.0.0.0/8', '169.0.0.0/24' ]
```

And you can specify the hashes in the ports section, as well.